Ds3/Calls/GetStorageDomainsSpectraS3Request.cs

```diff
@@ -233,6 +240,18 @@ public GetStorageDomainsSpectraS3Request WithPageStartMarker(string pageStartMar
             }
             return this;
         }
+        public GetStorageDomainsSpectraS3Request WithSecureMediaAllocation(bool? secureMediaAllocation)
+        {
+            this._secureMediaAllocation = secureMediaAllocation;
+            if (secureMediaAllocation != null) {
```

Comment:
the correct way to check if a Nullable object is not null is by doing "secureMediaAllocation.HasValue"